bin/test-without-stubs

```diff
@@ -83,4 +83,5 @@ trap cleanup EXIT INT TERM
 # Otherwise CI may falsely report a test as passed when it has failed.
 # Run lint which is configured by .golangci.yml at the root of the repo
 # and if it's successful, run tests
+export GO111MODULE=auto
```

Comment:
I would recommend moving this command above line 80 and adding a comment for it to explain why it is needed currently.